packages/boa/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "preinstall": "node tools/install-python.js && node tools/install-requirements.js",
     "postinstall": "npm run build",
-    "test": "tap ./tests/**/*.js",
+    "test": "nyc tap ./tests/**/*.js",
```

Comment:
Please keep using the tap only, we are still not ready for code coverage with nyc :(

Reply:
No need to coverage  code? then just using `tap --no-cov` will suffice, if using `nyc 14` will trigger an error, please see [--experimental-loader does not work](https://github.com/tapjs/node-tap/issues/673#issuecomment-629813258)

Reply:
Pipcook children projects are maintained by lerna, and we have to consider about code coverage from the whole projects.

And this PR is to add the esm support only, so IMO this change on build stuffs are not proper here :)

Reply:
Ok, but if left as is the CI run will report an error, I'm not sure if the new PR is needed to remove the code coverage first.

Reply:
We can first see what errors are encountered, and it seems that we don't add code coverage.

Reply:
See https://github.com/tapjs/node-tap/issues/673

Reply:
Thank you, after reading @coreyfarrell's comment, IMO we have to add `--no-cov` here.

Reply:
fixed